mypy/checkexpr.py

```diff
@@ -645,15 +649,48 @@ def check_call_expr_with_callee_type(self,
                                          callee_type: Type,
                                          e: CallExpr,
                                          callable_name: Optional[str],
-                                         object_type: Optional[Type]) -> Type:
+                                         object_type: Optional[Type],
+                                         member: Optional[str] = None) -> Type:
         """Type check call expression.
 
         The given callee type overrides the type of the callee
         expression.
+
+        The 'callable_name' and 'object_type' are used to call plugin hooks.
```

Comment:
Also `'member'` is used for plugin hooks only? Maybe be more explicit and say that they are *only* used for this purpose.